src/config/graphical-session/wayland.md

```diff
@@ -85,9 +85,14 @@ compositors, and is installed as a dependency for most of them. Its package is
 ## Configuration
 
 The Wayland API uses the `XDG_RUNTIME_DIR` environment variable to determine the
-directory for the Wayland socket. In order to securely set this variable, you
-need the `elogind` service installed and enabled as your [session
-manager](./session-management.md).
+directory for the Wayland socket.
+
+Install `elogind` as your [session manager](./session-management.md) to
+automagically setup `XDG_RUNTIME_DIR`.
```

Comment:
No matter how much I like that word, "automagically" is a bit too informal, unfortunately. Could use "automatically", instead.

Reply:
technically this is part of UNIX/computer culture :p
http://www.catb.org/jargon/html/A/automagically.html

nevertheless, I changed it